### PR TITLE
Add resource group info to query events

### PR DESCRIPTION
**Pull request by nezihyigitbasi**

### CODE REVIEW

File: presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java [hunk 1]
```diff
@@ -145,13 +155,32 @@ public void testNormalQuery()
...
 
-        MaterializedResult result = queryRunner.execute(session, "SELECT count(*) FROM lineitem");
+        MaterializedResult result = runQueryAndWaitForEvents("SELECT count(*) FROM lineitem", expectedEvents, session);
         long expectedCompletedPositions = (long) result.getMaterializedRows().get(0).getField(0);
 
         assertEquals(actualCompletedPositions, expectedCompletedPositions);
         assertEquals(queryCompletedEvent.getStatistics().getTotalRows(), expectedCompletedPositions);
     }
 
+    @Test
```
Comment (raghavsethi) [inline presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java]: Can we simply add a check for resource group into existing tests? Also, there's no need for event-client-test RG, let's just use the user.

Comment (nezihyigitbasi) [inline presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java]: sure, updated.

Comment (raghavsethi) [inline presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java]: Can't see the update..

File: presto-tests/src/test/resources/resource_groups_config_simple.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "maxQueued": 1000,
+      "subGroups": [
+        {
+          "name": "user-${USER}",
+          "softMemoryLimit": "1MB",
+          "maxRunning": 3,
+          "maxQueued": 3
+        },
+        {
+          "name": "event-client-test",
```
Comment (raghavsethi) [inline presto-tests/src/test/resources/resource_groups_config_simple.json]: Remove

File: presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java [hunk 3]
```diff
@@ -72,36 +81,38 @@ private void tearDown()
...
         assertEquals(queryCreatedEvent.getContext().getServerVersion(), "testversion");
         assertEquals(queryCreatedEvent.getContext().getServerAddress(), "127.0.0.1");
         assertEquals(queryCreatedEvent.getContext().getEnvironment(), "testing");
         assertEquals(queryCreatedEvent.getContext().getClientInfo().get(), "{\"clientVersion\":\"testVersion\"}");
         assertEquals(queryCreatedEvent.getMetadata().getQuery(), "SELECT 1");
 
-        QueryCompletedEvent queryCompletedEvent = getOnlyElement(events.getQueryCompletedEvents());
+        QueryCompletedEvent queryCompletedEvent = getOnlyElement(generatedEvents.getQueryCompletedEvents());
+        assertTrue(queryCompletedEvent.getContext().getResourceGroupName().isPresent());
+        assertEquals(queryCompletedEvent.getContext().getResourceGroupName().get(), "global.event-client-test");
```
Comment (raghavsethi) [inline presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java]: Make this the user group

**Comment by nezihyigitbasi** [comment]
updated

**Pull request merged**